src/base/BlockInterface.php

```diff
@@ -97,6 +97,20 @@ public function blockGroup();
      */
     public function getFieldHelp();
 
+    /**
+     * Set the block's {{luya\cms\models\NavItemPage}} object.
+     *
+     * @param NavItemPage $page The page object.
+     */
+    public function setPage($page);
```

Comment:
we should typehint the expect object setPage(NavItemPage $page)

Reply:
Thanks for code review!
I added the PHP type declarations as you suggested.